src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -90,7 +96,7 @@ In some cases, you may want or need to identify the corresponding view transitio
 
 Note that the `transition:name` can only be used once on each page. Set this manually when Astro can't infer a proper name itself, or for more fine control over matching elements.
```

Comment:
```suggestion
Note that the provided `transition:name` value can only be used once on each page. Set this manually when Astro can't infer a proper name itself, or for more fine control over matching elements.
```

Reading this, it sounded like you could only use the directive once per page, but we mean that value can only be used once.